### DISCUSSION TITLE
When running inside a docker container, what folder to persist as volume?

### DISCUSSION THREAD

**Discussion by langure** [LiveHelperChat/livehelperchat#2042] [Q&A]
I installed LiveHelperChat in a container and got it working correctly. Now I had to restart the server and I've found out that the docker container it was running inside of, did not have a volume assigned. I thought that everything was saved in the database, but when re-starting the same container the initial installation script is shown.

Therefore I am assuming that this installation script saves the database data somewhere, but since I did not provide a volume, it got erased. 

What folder should I maintain persistent in order to not lose the initial configuration regardless of how many times I restart the docker container?

Thanks!

**Comment by remdex** [comment]
You can take a look at this repo just. https://github.com/LiveHelperChat/docker-standalone
